cfme/tests/intelligence/reports/test_import_export_reports_widgets.py

```diff
@@ -15,56 +15,51 @@
 pytestmark = [pytest.mark.tier(1), test_requirements.report]
 
 
-@pytest.fixture()
-def widget_file(appliance):
-    yaml_name = "import_widget.yaml"
+def yaml_path(yaml_name):
+    """ Returns yaml path of the file with yaml_name name"""
+    yaml_name = "{}.yaml".format(yaml_name)
+
     try:
         fs = FTPClientWrapper(cfme_data.ftpserver.entities.reports)
         file_path = fs.download(yaml_name, os.path.join("/tmp", yaml_name))
     except (FTPException, AttributeError):
         logger.exception("FTP download or YAML lookup of %s failed, defaulting to local", yaml_name)
-        file_path = data_path.join("ui/intelligence/import_widget.yaml").realpath().strpath
+        file_path = (data_path.join("ui/intelligence/{}".format(yaml_name)).realpath().strpath)
```

Comment:
You shouldn't have to use `format`, you're using `join()`. The `yaml_name` should be passed as second argument to `join`.

Reply:
Really, shouldn't even be passing `ui/intelligence`, you should be calling `data_path.join('ui', 'intelligence', yaml_name)`